### DISCUSSION TITLE
Which directories will contain large files?

### DISCUSSION THREAD

**Discussion by TopBanana123** [Tzahi12345/YoutubeDL-Material#760] [Q&A] [answered]
Hopefully I never skipped over this, but I noticed in the Docker yaml there are 5 volumes

appdata and users I assume as small and should be local. audio and video I assume will be large, so I will mount on a NAS. What about subscriptions? What goes in there?

Just trying to decide what goes where

Thanks!

**Comment by Tzahi12345** [comment, marked as answer]
If you use multi-user mode, it will be the `users` folder that will contain all the files.

If not, it will be the following:
* video
* audio
* subscriptions

Hope this helps!
